homepage.php

```diff
@@ -1,73 +1,112 @@
+<?php
+		// Deliver to login page if access to this page with out login
+		session_start();
+		if(!isset($_SESSION['login_user'])){
+				header("location:login.php");
+				}
+		else{
+			$User = $_SESSION['login_user'];
+		}
+?>
+
 <!DOCTYPE html>
 <html lang = "en">
 	<head>
-		<title> Homepage</title>
+		<title>Homepage</title>
 		<meta chrset="utf-8"/>
 		<meta name="Keywords" content = "html5"/>
 		<meta name = "Author" content = "Tony Nguyen"/>
 		<meta name = "Description" content = "Music database"/>
 		<meta name = "viewport" content = "width=device-width. initial-scale=1"/>
 		<!--Connected to css -->
-		<link rel ="stylesheet" href = "style5.css"> 
+		<link rel ="stylesheet" href = "style1.css">
 		
 	</head>
 <body>
+	<!--Set for grid system for the page -->
 	<div class ="grid-container">
+		<!-- The div for nav and in the div it contain the logo and a function to called the nav -->
 		<div class="navigation">
+			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
 				<a href = "homepage.php"><img src = "images/logo8.png" height = "100"></a>
 			</div>
-			<nav>
-				<ul>
-					<li><a href = "homepage.php">Home</a></li>
-					<li><a href = "musicplayer.php">Music Play</a>
-						<ul>
-							<li><a href= "Query1.php">Query1</a></li>
-							<li><a href= "Query2.php">Query2</a></li>
-						</ul>
-					</li>
-					<li><a href = "login.php">Contact</a></li>
-					<li><a href = "login.php">LoginOut</a></li>
-					<li><a href = "signup.php">SignUp</a></li>
-				</ul>
-			</nav>
+			<?php
```

Comment:
I pull nav from the link from nav_login.php. to the login page so it can make it look or clear and easy to understand.